js/src/lib/Details/JSONLDItem.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { packageLink } from '../util';
+
+/**
+ * JSON-ld is a technology that allows crawlers to get info about
```

Comment:
Might be true, do you want me to remove it for now? 

Netlify does have pre-rendering though if we enable it https://www.netlify.com/docs/prerendering/